GetIntoTeachingApiTests/Services/StoreTests.cs

```diff
@@ -504,6 +504,54 @@ public async void SearchTeachingEvents_FilteredByStartBefore_ReturnsMatching()
                 options => options.WithStrictOrdering());
         }
 
+        [Fact]
+        public async void SearchTeachingEvents_FilteredByDefaultStatusId_ReturnsOpenEvents()
+        {
+            await SeedMockTeachingEventsAndBuildingsAsync();
+            var request = new TeachingEventSearchRequest()

```

Comment:
This isn't really testing the default as you're explicitly setting the `StatusId` to `Open` - I think it should just be:

```
var request = new TeachingEventSearchRequest();
```

and then you could have a test in `TeachingEventSearchRequest` that checks the default value for the model